test/awesome_map/BoundaryInterceptorSpec.js

```diff
@@ -633,5 +639,118 @@ define(function(require) {
+                    var targetState = getTargetState(10,0);
+                    interceptor.handleTransformStarted(null, { event: evt, targetState: targetState });
+                    expect(callback).not.toHaveBeenCalled();
+                });
+                it('when scrolling beyond the right boundary below the sensitivity threshold', function() {
+                    setCurrentTransformState(-100,0); // Top-right of content
+                    var targetState = getTargetState(-110,0);
+                    interceptor.handleTransformStarted(null, { event: evt, targetState: targetState });
+                    expect(callback).not.toHaveBeenCalled();
+                });
+                it('when scrolling beyond the bottom boundary below the sensitivity threshold', function() {
+                    setCurrentTransformState(0,-200); // Bottom of content
+                    var targetState = getTargetState(0,-210);
+                    interceptor.handleTransformStarted(null, { event: evt, targetState: targetState });
+                    expect(callback).not.toHaveBeenCalled();
+                });
+                it('when scrolling beyond the top boundary while viewport not at the top boundary of the content', function() {
+                    setCurrentTransformState(0,-200); // Bottom of content
+                    var targetState = getTargetState(0,30);
+                    interceptor.handleTransformStarted(null, { event: evt, targetState: targetState });
+                    expect(callback).not.toHaveBeenCalled();
+                });
+                it('when scrolling beyond the left boundary while viewport not at the left boundary of the content', function() {
+                    setCurrentTransformState(-100,0); // Top-right of content
+                    var targetState = getTargetState(30,0);
+                    interceptor.handleTransformStarted(null, { event: evt, targetState: targetState });
+                    expect(callback).not.toHaveBeenCalled();
+                });
+                it('when scrolling beyond the right boundary while viewport not at the right boundary of the content', function() {
+                    setCurrentTransformState(0,0); // Top-left of content
+                    var targetState = getTargetState(-130,0);
+                    interceptor.handleTransformStarted(null, { event: evt, targetState: targetState });
+                    expect(callback).not.toHaveBeenCalled();
+                });
+                it('when scrolling beyond the bottom boundary while viewport not at the bottom boundary of the content', function() {
+                    setCurrentTransformState(0,0); // Top of content
+                    var targetState = getTargetState(0,-230);
+                    interceptor.handleTransformStarted(null, { event: evt, targetState: targetState });
+                    expect(callback).not.toHaveBeenCalled();
```

Comment:
Same as I mentioned in ScrollListSpec, these tests could just call to a function that gets rid of the copypasta.  I like the getTargetStat and setCurrentTransformState, but you could take it a step further.